src/OrleansCounterControl/OrleansCounterControl.csproj

```diff
@@ -45,8 +45,24 @@
     <WarningsAsErrors>4014</WarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
It looks to me people arguing for using what Nuget is doing by default and avoid both extra work and go along with the tool. That being written, if I read this right, there should be a decision whether we should have soft references or not before moving to the new system and after that it doesn't matter. A point weighing in decision is if it matters in the meantime. Or did I stir up a hornet's nest now? :)